R/utils-custom.R

```diff
@@ -37,3 +37,11 @@ value_shape_pentagon <- function() 18
 label_emission_rank <- function() "Emission Rank"
 
 label_transition_risk <- function() "TR Score"
+
+label_bank <- function() "Bank"
+
+label_company <- function() "Company"
+
+label_tilt_sector <- function() "Tilt Sector"
+
+label_risk_categories <- function() "Risk Categories"
```

Comment:
``` r
# It's most maintainable to separate the content and formatting
label_tilt_sector <- function() "tilt sector"
label_tr_score <- function() "tr score"
label_transition_score <- function() "transition score"


# Format today
format_label <- tools::toTitleCase

label_tilt_sector() |> format_label()
#> [1] "Tilt Sector"
# Note "tr " may be a bad idea because it's unclear and hard to format
label_tr_score() |> format_label()
#> [1] "Tr Score"
# The fuller version is clearer and easier to format
label_transition_score() |> format_label()
#> [1] "Transition Score"

# Format tomorrow
format_label <- toupper

# The content of the labels doesn't change at all
label_tilt_sector() |> format_label()
#> [1] "TILT SECTOR"
label_tr_score() |> format_label()
#> [1] "TR SCORE"
label_transition_score() |> format_label()
#> [1] "TRANSITION SCORE"
```

<sup>Created on 2024-03-20 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>